src/Microsoft.AspNetCore.Razor.Tasks/RazorTagHelper.cs

```diff
@@ -51,6 +64,21 @@ protected override string GenerateResponseFileCommands()
             builder.AppendLine("-p");
             builder.AppendLine(ProjectRoot);
 
+            builder.AppendLine("-v");
+            builder.AppendLine(Version);
+
+            builder.AppendLine("-c");
+            builder.AppendLine(Configuration[0].GetMetadata(Identity));
+
+            for (var i = 0; i < Extensions.Length; i++)
+            {
+                builder.AppendLine("-n");
+                builder.AppendLine(Extensions[i].GetMetadata(Identity));
+
+                builder.AppendLine("-e");
+                builder.AppendLine(Path.GetFullPath(Extensions[i].GetMetadata(AssemblyFilePath)));
```

Comment:
Passing the name + assembly is mostly for clarity and diagnostics. It doesn't currently influence any behavior, but it feels wrong to me to leave it out.